packages/ui5-proxy-middleware/src/base/utils.ts

```diff
@@ -236,7 +240,7 @@ export async function resolveUI5Version(version?: string, log?: ToolsLogger): Pr
         ui5Version = version ? version : '';
         ui5VersionLocation = getYamlFile(process.argv);
     } else {
-        const minUI5Version = await getUI5VersionFromManifest(process.argv);
+        const minUI5Version = await getUI5VersionFromManifest(process.argv, manifest);
         ui5Version = minUI5Version && !isNaN(parseFloat(minUI5Version)) ? minUI5Version : '';
```

Comment:
I know this PR did not touch this line, but for such a simple piece of code this line is not very readable :)
`Number.isNan()` is a more robust version of the global `isNaN()` (source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN)

Reply:
I am sure whether this is really required because of `parseFloat`. Based on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN it is more robust when you provide strings, booleans etc, however, in our case we convert the input beforehand, therefore, it shouldn't be different.
So, it is something not to straight forward to change, therefore, I'd keep it our of here.